contracts/ethereum-sc/contracts/SharedStructs.sol

```diff
@@ -21,6 +21,6 @@ struct Deposit {
 struct Batch {
     uint256 nonce;
     uint timestamp;
-    uint lastUpdated;
+    uint lastUpdatedBlockNumber;
```

Comment:
`uint256` -- it's more verbose and recommended vs `uint` even though they effectively are the same thing.